### DISCUSSION TITLE
How to use @sqlc-vet-disable correctly?

### DISCUSSION THREAD

**Discussion by martinjanda** [sqlc-dev/sqlc#3503] [Q&A]
I use sqlc with sqlite. In `query.sql` I want select statement to FTS virtual table, so vet warns, that the relation does not exist.

I try to write `-- @sqlc-vet-disable` and `/* @sqlc-vet-disable */` but it doesn't work. 

How to use `@sqlc-vet-disable` in `query.sql`?

**Comment by Davincible** [comment]
@kyleconroy could you please shed some light on this? Also running into major issues because of this
